src/main/java/com/iexec/core/replicate/Replicate.java

```diff
@@ -64,7 +63,7 @@ public ReplicateStatus getCurrentStatus() {
     }
 
     @JsonIgnore
-    public Optional<ReplicateStatus> getLastRelevantStatus() {  // FIXME: remove Optional and add a no-args constructor
+    public ReplicateStatus getLastRelevantStatus() {
         // ignore cases like: WORKER_LOST and RECOVERING
 
         List<ReplicateStatus> statusList = getStatusUpdateList().stream()
```

Comment:
Not in the PR originally, but could we globally replace `getStatusUpdateList()` with direct access to `statusUpdateList` field ?

Reply:
Sure! Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/524/commits/dbcab6ac8f5b20da31a08674bd523f3c65bf1409.
(Issue with the commit name but that's the right one! :) )